docs/cluster-management/spot-instances.md

```diff
@@ -11,7 +11,7 @@ _WARNING: you are on the master branch, please refer to the docs on the branch t
 spot: false
 
 spot_config:
-  # additional instances with identical or better specs than the primary instance type (defaults to 2 instances sorted by price)
+  # additional instances with identical or better specs than the primary instance type (defaults to primary instance)
```

Comment:
how do you feel about "(defaults to no additional instances)" or "(defaults to only the primary instance)" to make it super clear?

Reply:
I like *"defaults to only the primary instance"* - this is one is super obvious. I'll apply this. The former can be interpreted I think.